src/supervisor.adoc

```diff
@@ -2270,6 +2282,18 @@ Invalid PTEs using a bounded timer, or making address-translation caches
 coherent with store instructions that modify PTEs.
 ====
 
+[[sec:svkt]]
+== "Svkt" Extension for Address-Independent Latency of User-Mode Translation of Supervisor Addresses, Version 0.2
```

Comment:
The Faults are what is constant time, not the Translation (Table Walks).
```suggestion
== "Svkt" Extension for Address-Independent Latency of User-Mode Faults of Supervisor Addresses, Version 0.2
```

Reply:
The translation and the faulting are hard to disentangle, since it’s the variable latency of the translation that made the latency of the fault vary. But your version might make the intent clearer; we can try it out.

“faults to” rather than “faults of”, right?

Reply:
Sure, "faults to".

Reply:
Done.